crosshair/diff_behavior.py

```diff
@@ -67,16 +75,16 @@ def describe(self, args_to_show: Set[str]) -> str:
 
 def describe_behavior(
     fn: Callable, args: inspect.BoundArguments
-) -> Tuple[object, Optional[str]]:
+) -> tuple[Any, None] | tuple[None, IgnoreAttempt]:
     with ExceptionFilter() as efilter:
         ret = fn(*args.args, **args.kwargs)
         return (ret, None)
     if efilter.user_exc is not None:
         exc = efilter.user_exc[0]
         debug("user-level exception found", repr(exc), *efilter.user_exc[1])
-        return (None, repr(exc))
+        return (None, IgnoreAttempt(exc))
```

Comment:
My apologies, I led you astray on this one: no need to wrap the user-level exception in IgnoreAttempt; I just wanted to change the "IgnoreAttempt" string below. And then the return type can just be `Tuple[object, Optional[BaseException]]` (note also that we use the imports from typing like `Tuple` instead of `tuple`, because, at the moment, I'm still supporting python 3.8)

Reply:
Thanks for clarifying. I reverted those changes.